nix/chainmain.nix

```diff
@@ -0,0 +1,8 @@
+{pkgs ? import ./default.nix {} }:
+let
+  released = (import (builtins.fetchTarball "https://github.com/crypto-org-chain/chain-main/archive/v3.1.1.tar.gz") { }).chain-maind;
```

Comment:
it'll be better to test with the current mainnet version (v2.1.1).

one possible extension test is to also have the v2->v3 upgrade on chain-main and check the IBC is still working, but that can perhaps be better done in a separate PR

Reply:
ok, i'll change to v2.1.1

Reply:
my mistake: 2.1.2 -- but I see it's already fixed: https://github.com/crypto-org-chain/cronos/pull/56/commits/258a63003458ec3269f23b3aaba2e67d0cc5abee